jython-processors/trigonometric-transformation/processor.json

```diff
@@ -2,13 +2,11 @@
 {
     "meta" : {
         // label: name of the data prep step as displayed, should be short
-        "label": "Custom processor polarize periodical feature",
```

Comment:
general comment: periodic seems to be more "English" than periodical. I would replace everywhere in plugin identifiers and labels.

"Periodic feature engineering" looks like a good choice. It has 77 million search results on Google, which is more than other searches I have tried.